common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -84,7 +86,9 @@
                 public Object run() {
                     try {
                         final Field unsafeField = Unsafe.class.getDeclaredField("theUnsafe");
-                        Throwable cause = ReflectionUtil.trySetAccessible(unsafeField);
+                        // We always want to try using Unsafe as the access still works on java9 as well and
+                        // we need it for out native-transports and many optimizations.
+                        Throwable cause = ReflectionUtil.trySetAccessible(unsafeField, false);
```

Comment:
Would the scary sounding message still be logged here for jdk9? Perhaps we can log a message for the user in addition to the code comments here.

Reply:
Nope... access to unsafe is handled special and so nothing is logged